catroid/src/main/java/org/catrobat/catroid/content/bricks/BroadcastMessageBrick.java

```diff
@@ -77,7 +77,7 @@ public View getView(Context context) {
 	}
 
 	@Override
-	public void onNewOptionSelected() {
```

Comment:
why are you changing also the Broadcastbrick

Reply:
it necessary to change the basefunction to enable the identification of multiplible dropdown's